Add additional info for OIDC endpoints (#1982)
buildkite/docs: pages/agent/v3/cli_oidc.md

<%= render "agent/v3/help/oidc_request_token" %>

> ## OIDC endpoints

Reviewer: Markdown style: Add a blank line after a heading.

```suggestion
## OIDC endpoints

```

> * OpenID Connect Discovery URL: https://agent.buildkite.com/.well-known/openid-configuration
> * JWKS URI: https://agent.buildkite.com/.well-known/jwks

Reviewer: Style: Bold the term for description lists.

```suggestion
* **OpenID Connect Discovery URL:** https://agent.buildkite.com/.well-known/openid-configuration
* **JWKS URI:** https://agent.buildkite.com/.well-known/jwks
```

<%= render "agent/v3/help/oidc_request_token" %>

## OIDC URLs

> If using a plugin, such as our [AWS assume-role-with-web-identity](https://github.com/buildkite-plugins/aws-assume-role-with-web-identity-buildkite-plugin) plugin, you'll need to provide an OpenID provider URL, the provider URL should be set to https://agent.buildkite.com.

Reviewer: General tightening. We can't always, but it's best to avoid "our" because it's unclear who it includes.

```suggestion
If using a plugin, such as the [AWS assume-role-with-web-identity](https://github.com/buildkite-plugins/aws-assume-role-with-web-identity-buildkite-plugin) plugin, you'll need to provide an OpenID provider URL. You should set the provider URL to:  https://agent.buildkite.com.
```

> For specific endpoints for OpenID or JWKS, see below.
> 
> ### Specific OIDC endpoints
> * OpenID Connect Discovery URL: https://agent.buildkite.com/.well-known/openid-configuration
> * JWKS URI: https://agent.buildkite.com/.well-known/jwks

Reviewer: Do you think this needs it's own heading? I think it could be simplified to:

```suggestion
For specific endpoints for OpenID or JWKS, use:

* **OpenID Connect Discovery URL:** https://agent.buildkite.com/.well-known/openid-configuration
* **JWKS URI:** https://agent.buildkite.com/.well-known/jwks
```
